starter-core/src/main/java/io/micronaut/starter/feature/function/template/http/httpFunctionJavaController.rocker.raw

```diff
@@ -0,0 +1,62 @@
+@import io.micronaut.starter.application.Project
+
+@args (
+Project project
+)
+
+@if (project.getPackageName() != null) {
+package @project.getPackageName();
+}
+
+
+import io.micronaut.http.annotation.*;
+import io.micronaut.http.annotation.Produces;
+import io.micronaut.http.MediaType;
+import io.micronaut.core.annotation.Introspected;
+
+@@Controller
+public class @project.getClassName()Controller {
+
+    @@Produces(MediaType.TEXT_PLAIN)
+    @@Get
+    public String index() {
+        return "Example Response";
+    }
+
+    @@Post
+    public SampleReturnMessage postMethod(@@Body SampleInputMessage inputMessage){
+      SampleReturnMessage retMessage = new SampleReturnMessage();
+      retMessage.setReturnMessage(String.format("Hello %s, thank you for sending the message", inputMessage.getName()));
```

Comment:
I think we should use String concatenation instead of `String.format`. I think concatenation is much faster.